Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/Dashboard/Driver.hs

```diff
@@ -160,6 +161,22 @@ type FleetStatsAPI =
     :> "fleet"
     :> Get '[JSON] Common.FleetStatsRes
 
+-------------------------------------------------------------------
+------- Send Sms to Driver ----------------------------------------
+
+type SendSmsToDriverViaDashboardAPI =
```

Comment:
What's the difference In SendSmsToDriverViaDashboardAPI and SendSmsToDriverAPI ?

Reply:
So In this dashboard use case, we need to track the volunteer sms hit count. So we require to increase the hit limit whenever the voluteer is able to successfully send the message.

But increasing the count is done in driver-app and driver app does not get the volunteer data. So,

for dashboard side following api is exponsed -` SendSmsToDriverViaDashboardAPI`
```
  Capture "driverId" (Id Common.Driver)
    :> "sendSms"
    :> ReqBody '[JSON] DDriver.SendSmsReq
    :> Post '[JSON] APISuccess
```

and for driver app side, as we require volunteer id as well, I am passing volunteer id from the apiTokenInfo internall by hitting this internal dashboard endpoint for driver-app.
 
```
type SendSmsToDriverAPI =
  Capture "driverId" (Id Common.Driver)
    :> Capture "volunteerId" Text
    :> "sendSms"
    :> ReqBody '[JSON] DDriver.SendSmsReq
    :> Post '[JSON] APISuccess
```